spec.yaml

```diff
@@ -921,6 +973,15 @@ paths:
           required: true
           schema:
             $ref: "./definitions.yaml#/definitions/V4Organization"
+          x-examples:
```

Comment:
I just double checked since I wasn't sure if this would work,
but the `api` doesn't support creating and adding users in one API call currently.

Reply:
I don't understand the point. Is that related to the example?

Reply:
Yeah the example block below, if I am right, refers to the `addOrganization` operationId.

it shows the `members` field in the example, populated with some sample members.

If you actually try to create an organisation like this, these members will not be added to the organization.

Reply:
OK. Created an issue: https://github.com/giantswarm/giantswarm/issues/3620